kolibri_explore_plugin/assets/src/views/ContentLightbox.vue

```diff
@@ -1,28 +1,25 @@
 <template>
 
-  <div class="content-lightbox">
-    <nav>
-      <UiToolbar
-        ref="toolbar"
-        :showIcon="true"
-      >
-        <template #icon>
+  <div ref="overlay" class="lightbox-overlay" @click="onOverlayClick($event)">
```

Comment:
The third commit is adding a new behavior: the lightbox can also be closed by clicking in the semi-transparent overlay. I personally like it, have you discussed this with Joana? In any case the 3rd commit should mention this new behavior.

Reply:
Oh yeah, I forgot that I added that. Hehe, force of habit - I've implemented a few of these lightbox things :b I'll ping her right now.